services/src/main/java/io/scalecube/services/methods/ServiceMethodRegistryImpl.java

```diff
@@ -36,10 +42,14 @@ public void registerService(
                                   Reflect.requestType(method));
 
                           // register new service method invoker
-                          methodInvokers.put(
-                              methodInfo.qualifier(),
-                              new ServiceMethodInvoker(
-                                  method, serviceInstance, methodInfo, errorMapper, dataDecoder));
+                          String qualifier = methodInfo.qualifier();
+                          ServiceMethodInvoker invoker = new ServiceMethodInvoker(
+                                  method, serviceInstance, methodInfo, errorMapper, dataDecoder);
+
+                          if (methodInvokers.containsKey(qualifier)) {
+                            LOGGER.warn("<{}> for API <{}> will be replaced by the following methodInvoker: <{}>", methodInvokers.get(qualifier), qualifier, invoker);
```

Comment:
This has to be `LOGGER.error()`  ;   use this string --
 `Existing methodInvoker ({}) for api '{}' is about to be replaced by new methodInvoker ({})`